stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/InitConnectionListener.kt

```diff
@@ -13,3 +17,5 @@ public abstract class InitConnectionListener {
 
     public data class ConnectionData(val user: User, val connectionId: String)
 }
+
+public data class ConnectionData(val user: User, val connectionId: String)
```

Comment:
Duplicate of the class nested in `InitConnectionListener`, so that we can more easily remove the entire `InitConnectionListener` later.

Reply:
Shouldn't we move it out of this file?
Maybe insdide of `models` package?